utilities/reduce_output_size.py

```diff
@@ -0,0 +1,26 @@
+import os
+import gzip
+import csv
+
+file_path = 'test_output.csv'
+
+# Check if the file exists
+if os.path.exists(file_path):
+    # Open the input and output files
+    with open(file_path, 'r') as infile, gzip.open('test_output.csv.gz', 'wt', newline='') as outfile:
+        reader = csv.DictReader(infile)
+
+        # Drop b_values columns
+        fieldnames = [field for field in reader.fieldnames if not field.startswith('bval_')]
+        writer = csv.DictWriter(outfile, fieldnames=fieldnames)
+        writer.writeheader()
+
+        columns_to_round = ['f', 'Dp', 'D', 'f_fitted', 'Dp_fitted', 'D_fitted']
+        
+        # Process each row
+        for row in reader:
+            filtered_row = {column: row[column] for column in fieldnames}
```

Comment:
Minor optimization, but it's probably faster to delete the columns you don't want rather than rebuilding the dictionary with columns you do want.

Reply:
I've made the requested changes. Could you please take another look and let me know if it is good to go?